content/intro-to-storybook/vue/es/deploy.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Desplegar Storybook'
+tocTitle: 'Desplegar'
+description: 'Desplegar Storybook online con GitHub y Netlify'
+---
+
+En este tutorial hemos ejecutado Storybook en nuestra máquina de desarrollo. También se puede compartir ese Storybook con el equipo, especialmente con los miembros no técnicos. Afortunadamente, es fácil implementar Storybook en línea.
+
+<div class="aside">
+<strong>¿Hiciste los test con Chromatic antes?</strong>
+<br/>
+🎉 Sus historias ya están desplegadas! Chromatic indexa de forma segura sus historias en línea y las rastrea a través de ramas y commits. Salta este capítulo y ve a la <a href="/react/es/conclusion">conclusión</a>.
```

Comment:
@IvanSotelo if you don't mind, update the link for the right conclusion, currently is pointing to the react version as opposed to the vue one.